translations/harbour-seaprint-de.ts

```diff
@@ -554,33 +554,33 @@
     </message>
     <message>
         <source>Ignore SSL errors</source>
-        <translation type="unfinished"></translation>
+        <translation>SSL Fehler ignorieren</translation>
     </message>
     <message>
         <source>In order to work with self-signed certificates of printers and CUPS instances, SSL errors needs to be ignored.</source>
-        <translation type="unfinished"></translation>
+        <translation>Um mit selbstsignierten Zertifikaten von Druckern und CUBS-Instanzen arbeiten zu können, müssen SSL-Fehler ignoriert werden.</translation>
     </message>
 </context>
 <context>
     <name>SupplyItem</name>
     <message>
         <source>Unknown type</source>
-        <translation type="unfinished"></translation>
+        <translation>unbekannter typ</translation>
     </message>
     <message>
         <source>Unnamed supply</source>
-        <translation type="unfinished"></translation>
+        <translation>unbenannter Toner</translation>
```

Comment:
Capitalization? This can be a lot more than toners (ink, fuser unit etc), supply is the generic name

Reply:
I left "Toner", as i am unsure of a word for this atm...